jwql/website/apps/jwql/oauth.py

```diff
@@ -177,46 +185,46 @@ def auth_info(fn):
 
     Returns
     -------
-    user_info : function
+    check_auth : function
         The decorated function
     """
 
-    def user_info(request, **kwargs):
-        """Store authenticated user credentials in a cookie and return
-        it.  If the user is not authenticated, store no credentials in
-        the cookie.
+    @auth_info
+    def check_auth(request, user, **kwargs):
+        """Check if the user is authenticated through ``auth.mast``.
+        If not, perform the authorization.
 
         Parameters
         ----------
         request : HttpRequest object
             Incoming request from the webpage
+        user : dict
+            A dictionary of user credentials
 
         Returns
         -------
         fn : function
             The decorated function
         """
 
-        cookie = request.COOKIES.get("ASB-AUTH")
+        # If user is currently anonymous, require a login
+        if user['ezid']:
 
-        # If user is authenticated, return user credentials
-        if cookie is not None:
-            response = requests.get(
-                'https://{}/info'.format(get_config()['auth_mast']),
-                headers={'Accept': 'application/json',
-                         'Authorization': 'token {}'.format(cookie)})
-            response = response.json()
+            return fn(request, user, **kwargs)
 
-        # If user is not authenticated, return no credentials
         else:
-            response = {'ezid' : None, "anon": True}
+            template = 'not_authenticated.html'
+            context = {'inst': '',
+                       'tools': MONITORS,
+                       'user': user,
+                       'version': jwql.__version__}
```

Comment:
This isn't really necessary, but thanks to the neato context processors you added, this could just be:
```python
context = {'inst': ''}
```